client/js/render.js

```diff
@@ -195,13 +196,19 @@ function renderChannelUsers(data) {
 }
 
 function renderNetworks(data, singleNetwork) {
+	const networks = JSON.parse(storage.get("networks")) || {};
+	data.networks.forEach((network) => networks[network.uuid] = networks[network.uuid] || {collapsed: false, uuid: network.uuid});
+	storage.set("networks", JSON.stringify(networks));
+
 	sidebar.find(".empty").hide();
 	sidebar.find(".networks").append(
 		templates.network({
 			networks: data.networks,
 		})
 	);
 
+	Object.keys(networks).forEach((key) => networks[key] && networks[key].collapsed && collapseNetwork($(`[data-uuid=${key}] button.collapse-network`)));
```

Comment:
Make this multi line please.

Also add `.network` to the selector.

Reply:
Fixed